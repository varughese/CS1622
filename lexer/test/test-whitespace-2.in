--a++<=4 > = 5/*88*/==71-1 +2 * 4//1<2;
a,b;c)d[e ]{ f}(
91+2
