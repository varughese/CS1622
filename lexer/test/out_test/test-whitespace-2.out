(1,SYM,"-")
(1,SYM,"-")
(1,NUM,"a")
(1,SYM,"+")
(1,SYM,"+")
(1,SYM,"<=")
(1,KEY,"4")
(1,SYM,">")
(1,SYM,"=")
(1,KEY,"5")
(1,SYM,"/*")
(1,KEY,"88")
(1,SYM,"*/")
(1,SYM,"==")
(1,KEY,"71")
(1,SYM,"-")
(1,KEY,"1")
(1,SYM,"+")
(1,KEY,"2")
(1,SYM,"*")
(1,KEY,"4")
(1,SYM,"/")
(1,SYM,"/")
(1,KEY,"1")
(1,SYM,"<")
(1,KEY,"2")
(1,SYM,";")
(2,NUM,"a")
(2,SYM,",")
(2,NUM,"b")
(2,SYM,";")
(2,NUM,"c")
(2,SYM,")")
(2,NUM,"d")
(2,SYM,"[")
(2,NUM,"e")
(2,SYM,"]")
(2,SYM,"{")
(2,NUM,"f")
(2,SYM,"}")
(2,SYM,"(")
(3,KEY,"91")
(3,SYM,"+")
(3,KEY,"2")
